.prettierignore

```diff
@@ -7,3 +7,4 @@ cover/
 .github/
 README.md
 .vscode/
+test/support/fixtures
```

Comment:
maybe we could ignore the whole `test` directory 🤔 what do you think? if you don't see any problem, please create a small PR on our elixir template too 🙏

Reply:
@andyduong1920 copied that

Reply:
@andyduong1920 After re-thinking, in my opinion, other files in the ```test``` folder should be run over the ```prettier``` file without ignoring but the *.json files should. If you also think it makes sense, I will create a PR

Reply:
So far we use prettier for only JS files and html.eex or heex file, for Json I think it's still having some value if we have prettier on it, might I know the reason we would like to ignore it @nvminhtue 🤔

Reply:
@andyduong1920 it's weird behavior because after running ```prettier``` alias command, it shows the below error.
![image](https://user-images.githubusercontent.com/42540264/176372749-453c23c5-089d-4ef5-9c35-839abf076064.png)

I found there is a plugin that is installed but never used ```prettier-plugin-eex```, what is the purpose?

Reply:
Does the `mix codebase.fix ` help to solve that @nvminhtue

Reply:
@andyduong1920 yes it does